lizmap/modules/lizmap/controllers/searchFts.classic.php

```diff
@@ -62,20 +69,23 @@ public function get()
         try {
             $lproj = lizmap::getProject($repository.'~'.$project);
             if ($lproj == null) {
-                jLog::log('The lizmap project '.$project.' does not exist !', 'lizmapadmin');
+                jLog::log('The lizmap project "'.$project.'" does not exist.', 'lizmapadmin');
```

Comment:
if `$lproj` is null and no `UnknownLizmapProjectException ` is thrown , it means a error from php while trying to get the project 
```suggestion
                jLog::log('Error while loading "'.$project.'" project', 'lizmapadmin');
```